pkg/operation/seed/seed.go

```diff
@@ -341,6 +342,11 @@ func BootstrapCluster(k8sGardenClient, k8sSeedClient kubernetes.Interface, seed
 		Tag:        &tag,
 	}
 
+	// Delete the mutatingwebhookconfigoration
```

Comment:
Please add a TODO which says that this logic can be removed in a future Gardener release.